components/dashboard/src/projects/NewProject.tsx

```diff
@@ -274,6 +274,10 @@ export default function NewProject() {
 
         const showSearchInput = !!repoSearchFilter || filteredRepos.length > 0;
 
+        const userLink = (r: ProviderRepository) => {
+            return `https://${new URL(r.cloneUrl).host}/${r.inUse?.userName}`
```

Comment:
nit: inconsistent `inUse?.userName` here, vs `inUse.userName` on line 331.

Reply:
This is because in line 331, the condition already checks whether `inUse` exists, therefore the `?` check is no longer needed.